Exiled.API/Extensions/MirrorExtensions.cs

```diff
@@ -212,16 +212,20 @@ public static void PlayCassieAnnouncement(this Player player, string words, bool
         public static void MessageTranslated(this Player player, string words, string translation, bool makeHold = false, bool makeNoise = true, bool isSubtitles = true)
         {
             StringBuilder annoucement = StringBuilderPool.Shared.Rent();
+
             string[] cassies = words.Split('\n');
             string[] translations = translation.Split('\n');
+
             for (int i = 0; i < cassies.Length; i++)
                 annoucement.Append($"{translations[i]}<size=0> {cassies[i].Replace(' ', ' ')} </size><split>");
 
+            string message = annoucement.ToString();
+
             foreach (RespawnEffectsController controller in RespawnEffectsController.AllControllers)
             {
                 if (controller != null)
                 {
-                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), annoucement, makeHold, makeNoise, isSubtitles);
+                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), message, makeHold, makeNoise, isSubtitles);
```

Comment:
```suggestion
                    SendFakeTargetRpc(player, controller.netIdentity, typeof(RespawnEffectsController), nameof(RespawnEffectsController.RpcCassieAnnouncement), StringBuilderPool.Shared.ToStringReturn(announcement), makeHold, makeNoise, isSubtitles);
```
(Remove `StringBuilderPool.Shared.Return(annoucement);` below)

Reply:
that code is executed inside `foreach`, so it is impossible to do like that, but i call `StringBuilderPool.Shared.ToStringReturn(announcement)` before to init `message` var